golang/testsuite/testsuite_impl/network_partition_test/network_partition_test_.go

```diff
@@ -173,15 +174,14 @@ func (test NetworkPartitionTest) addApiService(
 		networkCtx *networks.NetworkContext,
 		serviceId services.ServiceID,
 		partitionId networks.PartitionID,
-		datastoreSvc *datastore.DatastoreService) (*api_service_client.APIClient, error, ) {
-	configFactory := api.NewApiContainerConfigFactory(test.apiImage, datastoreSvc)
-	uncastedApiSvc, hostPortBindings, _, err := networkCtx.AddServiceToPartition(serviceId, partitionId, configFactory)
+		datastoreIPAddress string, datastorePort int) (*api_service_client.APIClient, error, ) {
```

Comment:
Same thing - one argument per line to match the rest

Reply:
yes